pycodestyle.cfg

```diff
@@ -0,0 +1,10 @@
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+[pycodestyle]
+max-line-length = 100
+indent-size = 4
+statistics = True
```

Comment:
Can we add either `W503`, `W504`, or both to the ignore list.  Personally I prefer line breaks before binary operators, but others may not.  Either way could be acceptable.